Loop/Managers/DeviceDataManager.swift

```diff
@@ -600,11 +602,11 @@ final class DeviceDataManager {
                         "## Device Communication Log",
                         deviceLogReport,
                         "",
-                        String(reflecting: self.watchManager!),
-                        "",
-                        String(reflecting: self.statusExtensionManager!),
-                        "",
-                        loopReport,
+                        // String(reflecting: self.watchManager!),
```

Comment:
Why are these being commented out? They're useful to see the state of watch and status manager.

Reply:
I was trying to reduce the size of the log file while increasing duration for pods.
I wrote up the sizes in thie [zulipchat conversation](https://loop.zulipchat.com/#narrow/stream/312259-Omnipod-DASH/topic/Log.20Files.20and.20Parser/near/269952885)

Reply:
Restored watch and status manager.